src/components/nodes/boundaryErrorEvent/index.js

```diff
@@ -28,6 +28,9 @@ export default {
       }),
     });
   },
+  validateIncoming() {
+    return false;
+  },
```

Comment:
Going with what @JCook21 said: The goal here is to have the other boundary events extend the base boundary event. So this method could be added only in `/src/components/nodes/boundaryEvent/index.js`, and, when we enable the other boundary events, they should be updated to extend that common base config.